contracts/KyberOasisReserve.sol

```diff
@@ -0,0 +1,273 @@
+pragma solidity 0.4.18;
+
+import "./ERC20Interface.sol";
+import "./Utils2.sol";
+import "./Withdrawable.sol";
+import "./KyberReserveInterface.sol";
+
+
+contract OtcInterface {
+    function getBuyAmount(ERC20 buyGem, ERC20 payGem, uint payAmt) public constant returns (uint fillAmt);
+    function sellAllAmount(ERC20 payGem, uint payAmt, ERC20 buyGem, uint minFillAmount) public returns (uint fillAmt);
+}
+
+
+contract TokenInterface is ERC20 {
+    function deposit() public payable;
+    function withdraw(uint) public;
+}
+
+
+contract KyberOasisReserve is KyberReserveInterface, Withdrawable, Utils2 {
+
+    uint constant internal ETH_SRC_AMOUNT = (10**18);
```

Comment:
1) this is DAI amount. 2) it is min dai src amount.

Reply:
called it MIN_TRADE_TOKEN_SRC_AMOUNT